.expeditor/cleanup_orphaned_test_resources.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+terminate_instance() {
+  local instance_id
+  instance_id="$1"
+
+  if aws --profile chef-cd ec2 terminate-instances --instance-ids "$instance_id"; then
+    echo "Terminated instance $instance_id"
+  else
+    echo "Failed to terminate instance $instance_id"
+  fi
+}
+
+terminate_instances() {
+  while read -r instance_id; do
+    terminate_instance "$instance_id"
+  done
+}
+
+echo "--- Setting up AWS credentials"
+export VAULT_UTIL_ACCOUNTS='{"aws":["chef-cd"]}'
+vault-util configure-accounts
```

Comment:
Can you remove the chef-cd AWS creds setup? Running this will interfere with chef-cd creds that expeditor already has setup. In the near future expeditor will run these jobs in a more optimal environment which will require this code again in order to request temporary chef-cd creds.

Reply:
should be taken care of
I left the `--profile chef-cd` to explicitly target the desired profile and simply removed the `vault-util` setup.